homeassistant/components/homematicip_cloud/sensor.py

```diff
@@ -65,6 +65,20 @@ def __init__(self, home):
         """Initialize access point device."""
         super().__init__(home, home)
 
+    @property
+    def device_info(self):
+        """Return device specific attributes."""
+        return {
+            'identifiers': {
+                # Serial numbers of Homematic IP device
+                (HMIPC_DOMAIN, self._device.id)
+            },
+            'name': self._device.label,
+            'manufacturer': 'eQ-3',
+            'model': self._device.modelType,
+            'sw_version': self._device.currentAPVersion,
+        }
```

Comment:
No via_hub here?

Reply:
No this is the hub.

Reply:
hub/ap device creation moved to __init__.py.

Only an additional sensor is added to AP here.